chapter3/log-sanitization/agent.py

```
@@ -100,7 +100,7 @@ def _chat_stream(self, messages):
             for chunk in stream:
                 yield chunk.get('message', {}).get('content', '')
         else:
-            # 用与 Ollama 相同的 JSON Schema 强约束输出结构（pii_values 数组），
+            # 用与 Ollama 相同的 JSON Schema 强约束输出结构（pii_items 数组），
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)